lib/Attachment.php

```diff
@@ -24,6 +24,7 @@
 
 namespace OCA\Mail;
 
+use function str_starts_with;
```

Comment:
https://www.php.net/manual/en/function.str-starts-with.php only exists in php8. we're not there yet.

Reply:
ah that's why static analysis was complaining